migrations/migrations.go

```diff
@@ -52,10 +51,10 @@ type Migrations []Migration
 
 // Options is the options for running migrations.
 type Options struct {
-	Db          basedb.Database
-	NodeStorage operatorstorage.Storage
-	DbPath      string
-	Network     beacon.Network
+	Db            basedb.Database
+	NodeStorage   operatorstorage.Storage
+	DbPath        string
+	NetworkConfig networkconfig.NetworkConfig
```

Comment:
As I mentioned previously. I think a separation between beacon and SSV is good, and where only beacon info is needed it makes sense to pass in the internal part of the config which holds the beacon.

Reply:
done